deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml

```diff
@@ -189,6 +189,13 @@ receivers:
         to: attributes["stream"]
         type: move
 
+{{ if .Values.sumologic.logs.container.keep_time_attribute }}
+      - id: move-time-attribute
+        from: body.time
+        to: attributes["time"]
+        type: move
+{{ end }}
+
```

Comment:
could you add a helm template test for that?

Reply:
:+1: done